static/scripts/asyncapi.yaml

```diff
@@ -10,62 +10,45 @@ info:
     [**Lemmy**](https://github.com/LemmyNet/lemmy) is a decentralized alternative to proprietary link aggregators such as Reddit.
     #### More information...
     - [Install your own server](https://join.lemmy.ml/docs/en/administration/administration.html)
-    - [WebSocket](https://join.lemmy.ml/docs/en/contributing/websocket_http_api.html#websocket)
-    - [Lemmy data types](https://join.lemmy.ml/docs/en/contributing/websocket_http_api.html#data-types)
-    - [Images](https://join.lemmy.ml/docs/en/contributing/websocket_http_api.html#images)
-    - [RSS/Atom feeds](https://join.lemmy.ml/docs/en/contributing/websocket_http_api.html#rss--atom-feeds)
-    - [Rate limiting](https://join.lemmy.ml/docs/en/contributing/websocket_http_api.html#default-rate-limits)
+    - [API reference](link_here)
```

Comment:
I spose this won't be necessary since this will be the API reference document.

Reply:
Yes. I've made a PR on lemmy-docs